packages/fxa-content-server/app/scripts/templates/sign_in_token_code.mustache

```diff
@@ -1,15 +1,15 @@
 <div id="main-content" class="card">
   <header>
-    <h1 id="fxa-signin-code-header">{{#unsafeTranslate}}Enter confirmation code<span class="description">for your Firefox account</span>{{/unsafeTranslate}}</h1>
+    <h1 id="fxa-signin-code-header" class="card-header">{{#unsafeTranslate}}Enter confirmation code<span class="card-header-span">for your Firefox account</span>{{/unsafeTranslate}}</h1>
```

Comment:
I can't see the styles for `.card-header-span` in this PR, but do we need a space before the opening `<span>` tag?

Reply:
We don't, it breaks to a new line.